notebooks/ExamplePlots.ipynb

```diff
@@ -0,0 +1,766 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot3.plot(xlog=True, linestyle='solid')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The histogram-style plots are an example of a plot using a different name for the preference settings, in this case `histo_prefs`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'xerrorbars': False,\n",
+       " 'yerrorbars': True,\n",
+       " 'ecolor': None,\n",
+       " 'capsize': None,\n",
+       " 'barsabove': False,\n",
+       " 'xlog': False,\n",
+       " 'ylog': False,\n",
+       " 'linestyle': 'None',\n",
+       " 'drawstyle': 'default',\n",
+       " 'color': None,\n",
+       " 'alpha': None,\n",
+       " 'marker': '.',\n",
+       " 'markerfacecolor': None,\n",
+       " 'markersize': None,\n",
+       " 'linecolor': None}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "plot.DataHistogramPlot.histo_prefs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Previously we explicitly set the error values, but we can also use one of the chi-square statistics to come up with error values. In this case it's just the square-root of the data value (so, for for $x \\sim 1$ bin, we have an error of 0):"
```

Comment:
Is there an extraneous 'for' in '(so, for for $x...'?

Reply:
@DougBurke-  I went ahead and tweaked two lines in ExamplePlots.ipynb:
    "(so, for for..."   -> "(so, for ..."
    "numper" -> "number"

Hope you don't mind.

Reply:
No problem